### DISCUSSION TITLE
RQ as backend?

### DISCUSSION THREAD

**Discussion by gabriels1234** [RealOrangeOne/django-tasks#28] [Q&A] [answered]
I heavily rely on RQ and Django-task-scheduler (formerly django-rq-scheduler).

Do you see the RQ worker itself implemented as a backend? It relies on forking to a new process for every task (preventing memory leaks). Also, what about simultaneous tasks? It would allow for further reducing costs.

Thanks!

**Comment by RealOrangeOne** [comment, marked as answer]
Yes, there could absolutely be an RQ-based backend. It wouldn't be something which would be integrated into Django itself, but it could happily live either within `django-tasks`, or in its own separate package.
